src/main/java/net/neoforged/neoforge/common/CommonHooks.java

```diff
@@ -827,6 +828,19 @@ public static FluidType getVanillaFluidType(Fluid fluid) {
         throw new RuntimeException("Mod fluids must override getFluidType.");
     }
 
+    public static boolean handleDripInfo(Fluid fluid, Level level, BlockPos pos) {
+        if (fluid.isSource(fluid.defaultFluidState()) && fluid.getFluidType().getDripInfo() != null) {
```

Comment:
I think it's probably better to get the fluid state from the level if possible. While this works for vanilla and basic forge fluids, a mod overriding this method could break. However, it seems as though the fluid is grabbed using `PointedDripstoneBlock#getFluidAboveStalactite`.

Reply:
I don't have a good way of doing this as the position passed into receiveStalactiteDrip is the position of the cauldron block. I don't think patching the method to include the fluid position or the fluidstate is a good idea as other cauldrons override the method and they would all have to be patched as well. Open to any other ideas though as usual